app/graphql/types/system.rb

```diff
@@ -5,6 +5,8 @@
 module Types
   # Definition of the System GraphQL type
   class System < Types::BaseObject
+    enforce_rbac Rbac::COMPLIANCE_VIEWER
```

Comment:
Umm, shouldn't this be `SYSTEM_READ`?

Reply:
I thought that if the system is coming form inventory, it should be part of the default view. :thinking:

Reply:
Then why do we have the `SYSTEM_READ` permission?

Reply:
Or is it the one that is being checked implicitly?

Reply:
Using `SYSTEM_READ` would be intuitive to me too, I've made the change to `COMPLIANCE_VIEWER` to be consistent with the comment to the API V1 `System` endpoint in the spreadsheet for RBAC feature permissions.